tools/vep_somatic_annotate_r93.cwl

```diff
@@ -91,13 +91,9 @@ inputs:
   cache_version: {type: int?, doc: "Version being used, should match build version", default: 93}
   ref_build: {type: string?, doc: "Genome ref build used, should line up with cache.", default: "GRCh38" }
 
-
 outputs:
   output_vcf:
     type: File
     outputBinding:
       glob: '*.vcf.gz'
-  output_tbi:
-    type: File  
-    outputBinding:
-      glob: '*.vcf.gz.tbi'
+    secondaryFiles: ['.tbi']
```

Comment:
I noticed this tool isn't being used anywhere else. Why are we using r93?

Reply:
Well, it's part of the annotation sub wf, which will soon be in all the call er sunworkflows. I wanted to use this tool to do two things:
1. Switch to using the publicly available (a benefit in itself) VEP cache, which is also indexed and smaller in terms of file size than the one we have been using 
2. To separate out the act of annotating from vcf2maf creation. These were in one docker before.